bftengine/include/bftengine/ReplicaFactory.hpp

```diff
@@ -0,0 +1,70 @@
+// Concord
+//
+// Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "Replica.hpp"
+#include "ReplicaImp.hpp"
+#include "ReadOnlyReplica.hpp"
+#include "ReplicaLoader.hpp"
+
+namespace preprocessor {
+class PreProcessor;
```

Comment:
Nit: class preprocessor::PreProcessor;

Reply:
Done

Reply:
Done

Reply:
It's not compiling:
`In file included from /concord-bft/bftengine/src/bftengine/ReplicaFactory.cpp:13:
/concord-bft/bftengine/include/bftengine/ReplicaFactory.hpp:20:21: error: no class named 'PreProcessor' in namespace 'preprocessor'
class preprocessor::PreProcessor;`

Reply:
So I left the previous version.